### PR TITLE
Remove client validation for loading certificate

### PR DESCRIPTION
**Pull request by Elsie4ever**
This pr is the second part of addressing the enrollment group create/update with certificate issue. I removed the validation on the client side to make sure we are not over validating the certificate content, so that way the certificate validation now is fully handled by service call. The file extension check is removed and I kept the help text in place to give user more guidance on the certificate format

---
This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

Thank you for contributing to the IoT extension!

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

Intent for Production

- [ ] It is expected that pull requests made to default or core branches such as `dev` or `main` are of production grade. Corollary to this, any merged contributions to these branches may be deployed in a public release at any given time. By checking this box, you agree and commit to the expected production quality of code.

Basic expectations

- [ ] If introducing new functionality or modified behavior, are they backed by unit and/or integration tests?
- [ ] In the same context as above are command names and their parameter definitions accurate? Do help docs have sufficient content?
- [ ] Have **all** the relevant unit **and** integration tests pass? i.e. `pytest <project root> -vv`. Please provide evidence in the form of a screenshot showing a succesful run of tests locally OR a link to a test pipeline that has been run against the change-set.
- [ ] Have linter checks passed using the `.pylintrc` and `.flake8` rules? Look at the CI scripts for example usage.
- [ ] Have extraneous print or debug statements, commented out code-blocks or code-statements (if any) been removed from the surface area of changes?
- [ ] Have you made an entry in HISTORY.rst which concisely explains your user-facing feature or change?

Azure IoT CLI maintainers reserve the right to enforce any of the outlined expectations.

A PR is considered **ready for review** when all basic expectations have been met (or do not apply).

### CODE REVIEW

**Comment by Elsie4ever** [comment]
image

**Review by vilit1 — APPROVED** [review]

**Comment by vilit1** [comment]
Add a history for the changes you did (specifically the fact that certificate files are no longer restricted by file extension)

**Comment by Elsie4ever** [comment]
@vilit1 history updated, please feel free to take another look, thanks!

File: HISTORY.rst [hunk 1]
```diff
@@ -26,6 +26,10 @@ unreleased
 
 * Fixed an issue with enrollement group certificate encoding 
 
+**IoT DPS updates**
+
+* Removed file extension restriction when user attach certificate for enrollment groups creation/update commands,
```
Comment (vilit1) [inline HISTORY.rst]: this affects individual enrollments too

Comment (vilit1) [reply HISTORY.rst]: "Removed file extension restriction for attached certificates in individual enrollments and enrollment groups creation/update commands"

Comment (Elsie4ever) [reply HISTORY.rst]: right, will update thanks!

**Pull request merged**